plugins/module_utils/user.py

```diff
@@ -323,6 +323,10 @@ def user_mod(cursor, user, host, host_all, password, encrypted,
                 update = True
 
             if update:
+                password_changed = True
+                msg = "Auth pugin updated"
+                if module.check_mode:
+                    return {'changed': True, 'msg': msg, 'password_changed': password_changed}
```

Comment:
```suggestion
                if module.check_mode:
                    return {'changed': True, 'msg': 'Auth plugin updated', 'password_changed': True}
```
I think it's OK to have the values here as it's not long